cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/DefaultFieldLineageReader.java

```diff
@@ -50,6 +50,14 @@ public DefaultFieldLineageReader(TransactionRunner transactionRunner) {
     });
   }
 
+  @Override
+  public Set<String> getDroppedFields(EndPoint endPoint, long start, long end) {
```

Comment:
This is not needed since you store everything in the summary, use the summary people can already tell what is dropped